lib/booking_bug/api.rb

```diff
@@ -0,0 +1,37 @@
+class BookingBug
+  class UnableToAuthenticate < StandardError; end
+
+  class Api
```

Comment:
I was planning to leave this until we get to the second implementation and look to refactor/scope these things as made sense then.

At this stage I am unsure of what otehr parts will exist or how they will stick together.